daprdocs/content/en/developing-applications/ides/visual-studio-code/_index.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Any reason the URL of this index has changed from vscode to visual-studio-code?

Reply:
I did that since I felt the full name of VS Code would benefit more people, I know VS Code is the official alias and resonates with us easily I felt the full name was more inclusive.

Reply:
Visual Studio code is the official name of the product. I think it may also help with SEO in the URL